.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Great job adding `test.md` to the `.gitignore`  file, however, please also add the `node_modules` directory as stated by the project requirements.